### PR TITLE
Flight category tutorials 

### PR DESCRIPTION
**Pull request by tsolakoua**

### CODE REVIEW

**Comment by tsolakoua** [comment]
Ready for review @minjikarin  :)

File: docs/resources/flights.md [hunk 1]
```diff
@@ -36,6 +36,47 @@ cheapest dates to fly:
     ]
 ```
 
+#### Search for destinations for a specific duration of stay
+
+For example let's say a traveler wants to spend 6 days in a city but doesn't have a strong preference for the destination. With the Flight Inspiration API we can recommend to the traveler the cheapest destinations based on the duration tine.
```
Comment (minjikarin) [inline docs/resources/flights.md:41]: duration tine ->duration time

File: docs/resources/flights.md [hunk 2]
```diff
@@ -314,16 +501,538 @@ only certain accredited parties can issue tickets. In the next section, we’ll
...
+            "14" 
+        ], 
+        "travelerIds": [ 
+            "1" 
+        ] 
+    } 
+} 
+```
+
+The `Flight Offers Price` API returns two bag offers for the given flight. The catalog shows that either one or two bags are available to be booked per passenger. Higher bag quantity will be rejected due to the airline's policy.
```
Comment (minjikarin) [inline docs/resources/flights.md:659]: higher bag 'quantities'

**Review by minjikarin — COMMENTED** [review]
Anna, this is a great informative page! the longer it is better! I only reviewed the structures and little typos but didn't read through the details. i think this is a good start already and we can add more and more details and adjust whenever we feel like it and whenever we receive questions from users.

**Comment by minjikarin** [comment]
tres bon!

**Pull request merged**